x-pack/plugins/actions/README.md

```diff
@@ -272,6 +242,160 @@ const result = await actionsClient.execute({
+| ----------------- | ------------------------------------------------------------------------------------------------------------------------- | ------------------- |
+| short_description | The title of the incident.                                                                                                | string              |
+| description       | The description of the incident.                                                                                          | string _(optional)_ |
+| externalId        | The id of the incident in ServiceNow. If presented the incident will be update. Otherwise a new incident will be created. | string _(optional)_ |
+| severity          | The name of the severity in ServiceNow.                                                                                   | string _(optional)_ |
+| urgency           | The name of the urgency in ServiceNow.                                                                                    | string _(optional)_ |
+| impact            | The name of the impact in ServiceNow.                                                                                     | string _(optional)_ |
+| category          | The name of the category in ServiceNow.                                                                                   | string _(optional)_ |
+| subcategory       | The name of the subcategory in ServiceNow.                                                                                | string _(optional)_ |
+
+#### `subActionParams (getFields)`
+
+No parameters for `getFields` sub-action. Provide an empty object `{}`.
+
+#### `subActionParams (getIncident)`
+
+| Property   | Description                           | Type   |
+| ---------- | ------------------------------------- | ------ |
+| externalId | The id of the incident in ServiceNow. | string |
+
+
+#### `subActionParams (getChoices)`
+
+| Property | Description                                                  | Type     |
+| -------- | ------------------------------------------------------------ | -------- |
+| fields   | An array of fields. Example: `[priority, category, impact]`. | string[] |
+
+---
+
+## Jira
+
+The [Jira user documentation `params`](https://www.elastic.co/guide/en/kibana/master/jira-action-type.html) lists configuration properties for the `pushToService` subaction. In addition, several other subaction types are available.
+### `params`
+
+| Property        | Description                                                                                                                                | Type   |
+| --------------- | ------------------------------------------------------------------------------------------------------------------------------------------ | ------ |
+| subAction       | The sub action to perform. It can be `pushToService`, `getIncident`, `issueTypes`, `fieldsByIssueType`, `issues`, `issue`, and `getFields` | string |
```

Comment:
```suggestion
| subAction       | The subaction to perform. It can be `pushToService`, `getIncident`, `issueTypes`, `fieldsByIssueType`, `issues`, `issue`, and `getFields`. | string |
```